### ISSUE TITLE
Arrow buttons in the "Edit Group" page clear all fields

### ISSUE THREAD

**Issue by wendelap** [OpenEnergyDashboard/OED#413]
When editing a group, if you select meters/groups in multiple of the 4 drop-down fields and then click any of the arrows, all 4 fields will be cleared, but only the field associated with the clicked button will have had its selected meters/groups attached to/disowned by the group. It might be easier to just use a single set of arrows for both meters and groups together.

**Comment by huss** [comment]
The option for an admin to select multiple meters and/or groups while editing the group should be going away with the resource generalization work. Thus, this issue should just go away. It is being left open until that happens.
